lib/Elastica/Param.php

```diff
@@ -138,9 +138,12 @@ public function setParams(array $params)
     public function addParam($key, $value)
     {
         if ($key != null) {
-            if (!isset($this->_params[$key])) {
-                $this->_params[$key] = array();
-            }
+            /**
```

Comment:
I check this in 5.4 http://sandbox.onlinephpfunctions.com/code/8dc8d94b0ccf05c77dffaf2c3b69bda702463f29